lib/gem_of/rake_tasks.rb

```diff
@@ -0,0 +1,226 @@
+require "fileutils"
+require "rototiller"
+require "yard"
+require "rubocop/rake_task"
+require "flog_task"
+require "flay_task"
+require "roodi_task"
+require "rubycritic/rake_task"
+
+module GemOf
+  # a bunch of instances of other rake tasks
+  # @example create an instance of me in your rakefile
+  #   GemOf::RakeTasks.new
+  # @api public
+  class RakeTasks
+    include Rake::DSL if defined? Rake::DSL
```

Comment:
If Rake::DSL isn't defined wouldn't everything else blow up? Should we just throw here instead?

Reply:
yeah.  this is copy pasta from somewhere.